cmd/wc/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func main() {
+	var numLines, numWords, numChars bool
+	flag.BoolVar(&numLines, "l", false, "print the number of lines")
+	flag.BoolVar(&numWords, "w", false, "print the number of words")
+	flag.BoolVar(&numChars, "c", false, "print the number of characters")
+
+	flag.Usage = func() {
+		fmt.Println("Usage: wc [options] [file]")
+		fmt.Println("Options:")
+		flag.PrintDefaults()
+	}
+
+	flag.Parse()
+
+	filePath := flag.Arg(0)
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		fmt.Println("No such file or directory")
+		os.Exit(1)
```

Comment:
log.Fatal with the error itself (it can be different than "No such file or directory)